src/ds5/ds5-options.cpp

```diff
@@ -15,11 +15,15 @@ namespace librealsense
             }
             case 1:
             {
-                return "On";
+                return "Laser";
             }
             case 2:
             {
-                return "Auto";
+                return "Auto Laser";
+            }
+            case 3:
+            {
+                return "LED";
             }
             default:
```

Comment:
PLs check that with D400 the control.get_range() is not affected and still reports [0..2] range